.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*  @antleblanc @cbourgois @frenautvh @jleveugle
```

Comment:
I suppose it is a copy/paste from existing rollupification, might need an update